public/src/actions/admin_questions.jsx

```diff
@@ -33,7 +33,27 @@ export function fetchAllQuestions() {
   });
 }
 
-export default function createQuestion(questionInfo, history) {
+export function fetchQuestion(id) {
+  return dispatch => axios.get(`/questions/${id}`, {
+    headers: { Authorization: ` JWT ${localStorage.getItem('auth_token')}` }
+  })
+  .then((res) => {
+    const questionData = res.data;
+    // TO-DO Find a better way to handle this data
+    const { responses } = questionData;
+    questionData.responses = responses ? questionData.responses.split(',') : '';
```

Comment:
since i'm returning questionData I wanted to make sure I changed responses there, maybe i'll remove the deconstruction, good point